server/settings/main.py

```diff
@@ -0,0 +1,11 @@
+from ayon_server.settings import BaseSettingsModel, SettingsField
+
+
+class MotionBuilderSettings(BaseSettingsModel):
+    workfile_test: bool = SettingsField(
+        title="Workfile Test")
```

Comment:
Admittedly I'd be very confused by seeing this in settings - I wouldn't know what to do with it. Nor do I feel it's really a "setting" for production or anything. Unless the tests generate a report that we want to request studios to run that are facing issues?

This is solely a development feature. Should we maybe just enable the test menu when e.g. `AYON_DEBUG` or whatever env var is set?

Reply:
This could be the option. If `AYON_DEBUG` is set to 1, it would show the testing menu.